### PR TITLE
updated docs and comments

### LINKED ISSUE #1349: Update python commands in documentation
**Description of issue or feature request**:
With the release of tuf 0.17.0 python2 support was removed. Parts of the documentations contain commands that state the usage of python instead of python3 and pip instead of pip3. The commands with "python" will not work outside the virtualenv. It would help to explicitly state "python3" and "pip3" here.

It would be helpful to agree on as to where and what changes might be appropriate.

**Current behavior**:
`pip install -r requirement-dev.txt`
`python aggregate_tests.py`

**Expected behavior**:
`pip3 install -r requirements-dev.txt`
`python3 aggregate_tests.py`

### PR DESCRIPTION
**Pull request by samuelgregorovic**
Fixes #1349 

updated docs and comments (exclude Python2 end-of-life):

Updated/removed documented commands and comments which were referencing Python2. Also updated links to documentation referencing Python2 docs (unchanged where needed)

Signed-off-by: Samuel Gregorovic <samuelgregorovic@gmail.com>

### CODE REVIEW

**Review by trishankatdatadog — APPROVED** [review]
LGTM, thanks!

**Comment by samuelgregorovic** [comment]
np :)

**Comment by joshuagl** [comment]
Thanks for taking the time to work on this @samuelgregorovic! I don't think we need to change tox.ini and the GitHub workflows, they are running in environments where python is a link to python3 already.

**Comment by samuelgregorovic** [comment]
@joshuagl i just thought it would be better for consistency as mentioned in the issue, even though in some places it was not necessary. thank you for the heads up anyway!

File: tuf/repository_tool.py [hunk 1]
```diff
@@ -629,7 +629,7 @@ def get_filepaths_in_directory(files_directory, recursive_walk=False,
 
     # FIXME: We need a way to tell Python 2, but not Python 3, to return
     # filenames in Unicode; see #61 and:
-    # http://docs.python.org/2/howto/unicode.html#unicode-filenames
+    # http://docs.python.org/howto/unicode.html#unicode-filenames
```
Comment (jku) [inline tuf/repository_tool.py:632]: I believe the whole comment here could be removed

**Review by jku — APPROVED** [review]
LGTM, left one nitpick but it's fine as is...

@joshuagl did you have real doubts about tox and the workflows or was it just a comment?

**Comment by jku** [comment]
There is a small conflict now though, would you mind resolving that?

**Comment by samuelgregorovic** [comment]
@jku done

**Comment by joshuagl** [comment]
> @joshuagl did you have real doubts about tox and the workflows or was it just a comment?

No real doubts, just cautious about changing things that don't need it. I think the consistency argument @samuelgregorovic made is a reasonable one.

Thanks for checking @jku and thanks for the patch @samuelgregorovic !

**Comment by jku** [comment]
So I intended to merge this. Then I noticed I had asked to "resolve" the conflict without suggesting rebase. Github UI makes it so easy to merge instead so samuel had done that ... and this creates a git history that is too complex for my small brain.

So I figured I'd rebase it myself, make a PR to get CI results (just in case), but now of course I can't merge that one because I can't review my own PR :facepalm: 

Soo, @joshuagl maybe easiest if you approve #1493 (content is identical to this one, just rebased instead of reverse-merged)?

**Comment by jku** [comment]
Merged in #1439, thanks @samuelgregorovic

**Comment by samuelgregorovic** [comment]
@jku np

**Pull request closed without merging**